### ISSUE TITLE
GitHub allows a contributor to sign CCLA/ICLA for PR created by other contributors

### ISSUE THREAD

**Issue by ashav-github** [communitybridge/easycla#2475]
Steps:

1. Login to Github with your credential and browse a repo(you should not be the repo owner) where PR is created by other contributors who is gated by EasyCLA
2. Click on the PR created by other contributor and gated.
3. Click on 'Not Covered' button and complete the CCLA/ICLA sign-in process.

Actual Result: Contributor is able to sign CCLA/ICLA for other contributor's PR.

Expected Result: Github/EasyCLA should not allow to sign CCLA/ICLA for other Contributor.

**Comment by vinod-kadam** [comment]
As discussed with @pranab-bajpai & @dealako, this is same behavior as  V1. So, marked as Post-EasyCLAv2Release.

**Comment by dealako** [comment]
Won't fix. Out of scope of current product requirements/features.
